contrib/extensions/sling-pipes/src/main/java/org/apache/sling/pipes/internal/PlumberServlet.java

```diff
@@ -47,13 +48,17 @@
  * it can also be launched against a container pipe resource directly (no need for path parameter)
  *
  */
-@SlingServlet(resourceTypes = {Plumber.RESOURCE_TYPE,
-        ContainerPipe.RESOURCE_TYPE,
-        AuthorizablePipe.RESOURCE_TYPE,
-        WritePipe.RESOURCE_TYPE,
-        SlingQueryPipe.RESOURCE_TYPE},
-        methods={"GET","POST"},
-        extensions = {"json"})
+@Component(service = {Servlet.class},
+        property= {
+                "sling.servlet.resourceTypes=" + Plumber.RESOURCE_TYPE,
```

Comment:
You can use the constants from o.a.s.servlets.resolver.internal.ServletResolverConstants or the new ones in Sling API 2.16.0 (see https://issues.apache.org/jira/browse/SLING-6249)